flamingpy/cv/ops.py

```diff
@@ -32,6 +32,9 @@ def SCZ_mat(adj):
         adj (array): N by N binary symmetric matrix. If modes i and j are
             linked by a CZ, then entry ij and ji is equal to the weight of the
             edge (1 by default); otherwise 0.
+    Keyword Args:
```

Comment:
I'm not sure that the "Keyword Args" heading appears anywhere else.

Reply:
OK, I'll change to just "Args".